src/layout.hoc.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import React from 'react'
+import { compact, getDisplayName } from './utils'
+import { SIZE } from './values'
+
+function getSize(size) {
```

Comment:
Missing types, though not sure if we want to set a linting rule to require types. Flow might have good enough type inference here to know that the param has to be a Symbol for example.